lib/src/m_login.dart

```diff
@@ -290,6 +290,58 @@ class MLogin {
     );
   }
 
+  /// This function is used to give an overview of the children the user
+  /// has added to her M-Login account.
+  ///
+  /// On this page, new children can be added. Existing children are displayed
+  /// and their details can be viewed and edited via links to the respective
+  /// detail pages.
+  /// For direct entry on the detail page, use [openChildDetailPage].
+  ///
+  /// Returns [true] in case the user finishes the page using the `done` button,
+  /// [false] in any other case (e.g., the user pressed the "cancel" button in
+  /// the iOS browser, or the back button on Android). This does not infer any
+  /// data change or validation and can safely be ignored.
```

Comment:
Meinst du damit, dass wenn der user nicht den fertig button nutzt, die Änderungen beim M-Login nicht persistiert werden? Ich glaub das stimmt nicht, oder? Also zumindest der Foto upload, wenn du den erfolgreich gemacht hast, ist das Foto ja da, egal was du danach machst?

Reply:
Nö eigentlich mein ich dass der Wert nix aussagt. Aber vielleicht kann mans noch besser formulieren. Das ist halt ein Absatz der bei den meisten Funktionen so steht, wenn dann sollte ichs bei allen anpassen.

Reply:
```suggestion
  /// Returns [true] in case the user finishes the page using the `done` button,
  /// [false] in any other case (e.g., the user pressed the "cancel" button in
  /// the iOS browser, or the back button on Android)
  ///
  /// Note that this does not contain any information whether data was changed
  /// or validated. It's safe to ignore the returned value and just assume data
  /// was changed.
```

Reply:
Was hältst du davon?

Reply:
Hab nochmal die Doc angepasst und den Text etwas exakter gemacht, und an den Stellen wo es sich gleich verhält die Doc vereinheitlicht.